Sources/Core/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(Core
+  FileSystem.swift)
```

Comment:
I think with Mattt's cleanup we've started using the `+Extensions` suffix for files that just have extensions for existing types.  Makes it easier to pick out the core implementation in file lists, Open Quickly UI, etc.